src/test/java/uk/gov/companieshouse/chd/order/api/controller/MissingImageDeliveriesControllerTest.java

```diff
@@ -47,20 +50,23 @@
 	@Test
 	@DisplayName("Create Missing image delivery successfully")
 	public void createMissingImageDeliverySuccessfully() {
-		final ResponseEntity<MissingImageDeliveriesDTO> response = controllerUnderTest.createMissingImageDelivery(
+
+		when(createMissingImageDeliveryItemRequestValidator.getValidationErrors(MISSING_IMAGE_DELIVERIES_DTO))
+			.thenReturn(new ArrayList<String>());
+		final ResponseEntity<Object> response = controllerUnderTest.createMissingImageDelivery(
 				MISSING_IMAGE_DELIVERIES_DTO, request);
-		
 		assertThat(response.getStatusCode(), is(HttpStatus.CREATED));
-		assertEquals(MISSING_IMAGE_DELIVERIES_DTO.getCompanyName(), response.getBody().getCompanyName());
-		assertEquals(MISSING_IMAGE_DELIVERIES_DTO.getCompanyNumber(), response.getBody().getCompanyNumber());
-		assertEquals(MISSING_IMAGE_DELIVERIES_DTO.getFilingHistoryType(), response.getBody().getFilingHistoryType());
-		assertEquals(MISSING_IMAGE_DELIVERIES_DTO.getFilingHistoryCategory(), response.getBody().getFilingHistoryCategory());
-		assertEquals(MISSING_IMAGE_DELIVERIES_DTO.getFilingHistoryDate(), response.getBody().getFilingHistoryDate());
-		assertEquals(MISSING_IMAGE_DELIVERIES_DTO.getFilingHistoryDescription(), response.getBody().getFilingHistoryDescription());
-		assertEquals(MISSING_IMAGE_DELIVERIES_DTO.getId(), response.getBody().getId());
-		assertEquals(MISSING_IMAGE_DELIVERIES_DTO.getItemCost(), response.getBody().getItemCost());
-		assertEquals(MISSING_IMAGE_DELIVERIES_DTO.getOrderedAt(), response.getBody().getOrderedAt());
-		assertEquals(MISSING_IMAGE_DELIVERIES_DTO.getPaymentReference(), response.getBody().getPaymentReference());
 	}
```

Comment:
Why have these assertions been removed?

Reply:
These were removed after speaking to Hywel, they were put in place to cover sonar for the getters. The response entity type was also changed to object and the assertions were no longer working. The getters are now checked during the validation test

Reply:
OK, well I think the correctness of the response is asserted in the `MissingImageDeliveryControllerIntegrationTest` now anyway. 👍